hc/front/views.py

```diff
@@ -330,6 +331,7 @@ def channel_checks(request, code):
         "channel": channel
     }
 
+    print("returned page")
```

Comment:
I believe this print statement was introduced for debugging purposes. In which case you should avoid pushing code with debug statements.